cdap-ui/server/aggregator.js

```diff
@@ -211,6 +211,11 @@ function stripResource(key, value) {
 function emitResponse(resource, error, response, body) {
   var timeDiff = Date.now() - resource.startTs;
 
+  if (response && (response.statusCode === 401 || response.statusCode === 403)) {
```

Comment:
Should we do this on a `403`? `403` should mean the client is authenticated, but attempted to access a resource they don't have permissions on.

Reply:
Fair point. I will check only for 401 status 👍